app/models/user.rb

```diff
@@ -1,16 +1,35 @@
 class User < ActiveRecord::Base
   include Sluggable
   has_secure_password validations: false
-  has_many :reviews
+
+  has_many :follower_connections, class_name: 'Connection', foreign_key: 'guide_id'
+  has_many :following_connections, class_name: 'Connection', foreign_key: 'follower_id'
```

Comment:
If you have `guide` and `follower` then this should be `guide_connections` if you have `follower_connections` ?